rs/tests/system_tests.bzl

```diff
@@ -129,6 +130,7 @@ def system_test(
 
     Args:
       name: base name to use for the binary and test rules.
+      variant: string to identify the variant of this test.
```

Comment:
what's a variant? what are the possible values? if it's only used in naming, why not simply use `name_<variant>` as a `name` when calling the `system_test` and reduce complexity?